Update runtime to 42

X-m7 opened the pull request:
Also removed libadwaita and its build dependencies since that is now included in the runtime.

Currently runtime 40 is used, which has been marked end-of-life since 21 March 2022.

X-m7 committed:
Update runtime to 42

Also updated libsass and sassc, and removed libadwaita since that is now
included in the runtime.

hfiguiere commented:
> Also updated libsass and sassc, and removed libadwaita since that is now included in the runtime.

I think both libsass and sassc where used to build libadwaita, which mean they would become unecessary...

X-m7 committed:
Remove libsass and sassc

Those were only necessary to build libadwaita apparently.

X-m7 commented:
> > Also updated libsass and sassc, and removed libadwaita since that is now included in the runtime.
> 
> I think both libsass and sassc where used to build libadwaita, which mean they would become unecessary...

Yep, you're right, got rid of those and the app still builds and works for me, thanks!

X-m7 commented:
bot, build

SeaDve commented:
Thanks!
